lib/rules/prefer-node-protocol.js

```diff
@@ -4,12 +4,22 @@
  */
 "use strict"
 
-const { isBuiltin } = require("node:module")
 const getConfiguredNodeVersion = require("../util/get-configured-node-version")
 const getSemverRange = require("../util/get-semver-range")
 const visitImport = require("../util/visit-import")
 const visitRequire = require("../util/visit-require")
 const mergeVisitorsInPlace = require("../util/merge-visitors-in-place")
+const {
+    NodeBuiltinModules,
+} = require("../unsupported-features/node-builtins.js")
```

Comment:
nits: we could move `node-builtins.js` to outer - it's now not just used by those rules. :)

Reply:
Where do you think we should put the directory?

I was also thinking about going through and merging the [`no-deprecated-api`](https://github.com/eslint-community/eslint-plugin-n/blob/master/lib/rules/no-deprecated-api.js#L27) into the `[`unsupported-features`](https://github.com/eslint-community/eslint-plugin-n/tree/master/lib/unsupported-features)` so its all in one place as a consistent data set 🤔

Reply:
> Where do you think we should put the directory?

maybe `/lib/dataset/`?

> I was also thinking about going through and merging the [no-deprecated-api](https://github.com/eslint-community/eslint-plugin-n/blob/master/lib/rules/no-deprecated-api.js#L27) into the [unsupported-features](https://github.com/eslint-community/eslint-plugin-n/tree/master/lib/unsupported-features)

It would be nice if these rules could share data, so we don't have to worry about updating one and forgetting the other.